root/cdtoc/attach_list.tt

```diff
@@ -23,11 +23,14 @@
       <tr[% ' class="even"' IF zebra % 2 == 0 %]>
         <td class="pos"></td>
         <td>
-          <label
-            [%~ IF !medium.may_have_discids %] title="[% l('This medium format cannot have a disc ID attached') %]"[% END ~%]
-            [%~ IF this_medium_has_cdtoc %] title="[% l('This CDTOC is already attached to this medium') %]"[% END ~%]
-          >
-            <input type="radio" name="medium" value="[% medium.id %]"[% IF !medium.may_have_discids OR this_medium_has_cdtoc %] disabled="disabled"[% END %]/>
+          <label>
+            [% IF this_medium_has_cdtoc %]
+              <div class="cannot-attach-discid icon img" title="[% l('This CDTOC is already attached to this medium.') %]"></div>            
+            [% ELSIF !medium.may_have_discids %]
+              <div class="cannot-attach-discid icon img" title="[% l('This medium format cannot have a disc ID attached.') %]"></div>
```

Comment:
Please add the missing full stop to the same sentence on line 41, in a separate commit.